docs/pages/development/create-development-builds.mdx

```diff
@@ -0,0 +1,175 @@
+---
+title: Create development builds
+---
+
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+import { Terminal } from '~/ui/components/Snippet';
+import { Tab, Tabs } from '~/ui/components/Tabs';
+import { BoxLink } from '~/ui/components/BoxLink';
+
+Development builds can be created with [EAS Build](/build/introduction/) or locally on your computer if you have Xcode and Android Studio. In the following sections, you will find information on how to create a development build and then install them on an emulator/simulator or a physical device to continue developing your app.
+
+## Prerequisites
+
+You need to initialize your project with Expo. If you still need to do so, see the [initializing steps](/get-started/create-a-new-app/#initializing-the-project) if you are starting a new project.
+
+## Install expo-dev-client
+
+To initialize a development build, you need to install the `expo-dev-client` library in your project:
+
+<Terminal cmd={['$ npx expo install expo-dev-client']} />
+
+## Build with EAS
+
+We recommend EAS Build to manage your native projects. It allows a smooth experience, especially if you do not have experience with Xcode and Android studio builds or have them installed locally on your computer.
+
+### Configure the project
+
+To get started, you need to configure your project for EAS Build by running the command:
+
+<Terminal cmd={['$ eas build:configure']} />
```

Comment:
This is no longer required, eas build will create for you if you don't have it already